en/installation/docker.md

```diff
@@ -1,7 +1,7 @@
 # Using Docker for local development
 
 You can use Docker to have a local CONSUL installation for development if:
-- You're having troubles having [prerequisites](prerequisites) installed.
+- You're having troubles having [prerequisites](prerequisites.md) installed.
```

Comment:
Correct me if I'm wrong; I thought the link was supposed to point to `../local_installation/prerequisites` :thinking:

Reply:
There is no such path in the folder structure 🤔 .
The file that references the local installation prerequisites is located inside the installation folder.
I think it's good that this link

Reply:
I thought the link didn't reference the folder structure but the URL structure :thinking:. Maybe there's some magic in git docs I don't know about. We'll check after merging this pull request.

Reply:
Apparently you're right and the link works properly this way :smile:.